docs/Makefile

```diff
@@ -28,7 +28,7 @@ help:
 	@echo "  doctest   to run all doctests embedded in the documentation (if enabled)"
 
 loop:
-	python watcher.py 'make html' `find .. -name '*.rst' -o -name '*.py'`
+	python watcher.py 'make html SPHINXOPTS=-nW' `find .. -name '*.rst' -o -name '*.py'`
```

Comment:
Added this, otherwise the test failures due to SPHINXOPTS=-nW aren't picked up locally.